### PR TITLE
README: add 'Building an AppImage on Linux'

### PR DESCRIPTION
**Pull request by asashnov**
README: add 'Building an AppImage on Linux' section.

### CODE REVIEW

**Comment by kelson42** [comment]
@mgautierfr Can you please review and merge this old PR?

File: README.md [hunk 1]
```diff
@@ -167,6 +167,27 @@ Kiwix-build.py will create several directories:
...
+virtualenv -p python3 ./
+source bin/activate
+pip install .
+hash -r
+export SKIP_BIG_MEMORY_TEST=1
+kiwix-build --target-platform native_dyn kiwix-desktop
+./scripts/create_kiwix-desktop_appImage.sh
+```
+The result: ~/kiwix-build-master/Kiwix--x86_64.AppImage
+
```
Comment (mgautierfr) [inline README.md:189]: There is already a documentation about how install kiwix-buidl in `prerequisite` section. And how to build kiwix-desktop in the `compilation` section.
All this new documentation could be resumed as: `Compile locally kiwix-desktop on native_dyn platform, then run the script create_kiwix-desktop_appImage.sh` (better written obviously)

**Review by mgautierfr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (asashnov) [reply README.md]: It wasn't obvious to me, as for a new developer on the project. Also, libzim tests failed on virtual machine with limited resources.

In updated PR the commands are visually compressed under the spoiler:

Screenshot from 2022-03-14 14-24-53

Comment (mgautierfr) [reply README.md]: The problem is that the information you give is not relative to creating a appimage. The only relevant information (to create a appimage) is that we must run (and how) `create_kiwix-desktop_appImage.sh` once kiwix-desktop is build.

The other information are useful, but are not related to appimage. We should/may keep them, but probably move them elsewhere (compilation of kiwix-desktop, troubleshooting section, ....)

**Pull request closed without merging**